routes/web.php

```diff
@@ -202,7 +202,7 @@
     Route::resource('roles', RoleController::class)->except(['show']);
 
     // Permissions
-    Route::resource('permissions', PermissionController::class)->except(['store', 'update', 'destroy']);
+    Route::resource('permissions', PermissionController::class, ['except' => ['store', 'update', 'destroy']]);
```

Comment:
last time you did some changes to routes which broke them, it was missing route naming, ive already fixed that issue, i think we will stick with the idea of each controller have only one method, we consider working with livewire controllers to make things happen.